rts/Lua/LuaSyncedRead.cpp

```diff
@@ -6236,20 +6238,18 @@ int LuaSyncedRead::GetUnitCurrentCommand(lua_State* L)
  * Same as `Spring.GetCommandQueue`
  *
  * @param unitID integer
- * @param count integer Number of commands to return, `-1` returns all commands, `0` returns command count.
+ * @param count integer Maximum amount of commands to return, `-1` returns all commands.
  * @return Command[] commands
  */
 /***
  * Get the count of commands for a unit.
  *
+ * @deprecated
```

Comment:
@rhys-vdw What workflow should I trigger "to test docgen" in this PR?

Reply:
https://github.com/beyond-all-reason/RecoilEngine/actions/runs/15031592986/job/42245068982?pr=2268

It's already running automatically if you touch any `rts/Lua/**/*`.